code/modules/projectiles/updated_projectiles/guns/flamer.dm

```diff
@@ -352,6 +354,145 @@
 		if(!current_mag || !current_mag.current_rounds)
 			return
 
+/obj/item/weapon/gun/flamer/M240T/examine(mob/user)
+	..()
+	to_chat(user, "<span class='notice'>Its hydro cannon contains [get_water()]/[max_water] units of water!</span>")
+
+//Returns the amount of water in the M240T
+/obj/item/weapon/gun/flamer/M240T/proc/get_water()
+	return reagents.get_reagent_amount("water")
```

Comment:
Does this really require a proc?
How about `#define M240T_WATER_AMOUNT reagents.get_reagent_amount("water")`?